.github/workflows/pr.yml

```diff
@@ -0,0 +1,52 @@
+# Copyright 2022 Google LLC
```

Comment:
nit: 2024?

ok - i see you're just moving this, and not added net-new code. nevermind!